CMake error encountered following *4.5.4. Exercise: Random Testing* in MathComponent Tutorial

| | |
|:---|:---|
|**_F´ Version_**| v3.0.1 |
|**_Affected Component_**| Ref/MathSender |
---
## Problem Description

Following steps in [4.5.4. Exercise: Random Testing](https://nasa.github.io/fprime/Tutorials/MathComponent/Tutorial.html#The-MathSender-Component_Write-and-Run-Unit-Tests_Exercise-Random-Testing), running `fprime-util check` encounters following CMake error,

```
/home/captainpike/repo/fork/fprime/Ref/MathSender/test/ut/main.cpp:2:10: error: #include expects "FILENAME" or <FILENAME>
    2 | #include STest/Random/Random.hpp
      |          ^~~~~
/home/captainpike/repo/fork/fprime/Ref/MathSender/test/ut/main.cpp: In function ‘int main(int, char**)’:
/home/captainpike/repo/fork/fprime/Ref/MathSender/test/ut/main.cpp:16:3: error: ‘STest’ has not been declared
   16 |   STest::Random::seed();
      |   ^~~~~
make[3]: *** [Ref/MathSender/CMakeFiles/Ref_MathSender_ut_exe.dir/build.make:106: Ref/MathSender/CMakeFiles/Ref_MathSender_ut_exe.dir/test/ut/main.cpp.o] Error 1
make[2]: *** [CMakeFiles/Makefile2:33363: Ref/MathSender/CMakeFiles/Ref_MathSender_ut_exe.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:33370: Ref/MathSender/CMakeFiles/Ref_MathSender_ut_exe.dir/rule] Error 2
make: *** [Makefile:11940: Ref_MathSender_ut_exe] Error 2
[ERROR] CMake erred with return code 2
```

See https://github.com/capsulecorplab/fprime/commit/21e104b29677f5c402c3fe63852faaf706643279

## How to Reproduce

### Setup/configuration
On Ubuntu 20.04 with python3,
1. `git clone git@github.com:capsulecorplab/fprime.git`
2. `cd fprime`
3. `python3 -m venv fprime-venv`
4. `. ./fprime-venv/bin/activate`
5. `pip install --upgrade fprime-tools fprime-gds`
6. `git checkout 21e104b`

### Build steps
In fprime repo with virtual environment activated, 
1. `cd Ref`
2. `fprime-util generate --ut`
3. `cd MathSender`
4. `fprime-util build --ut`
5. `fprime-util check`

## Expected Behavior

Unit tests should build without errors.

--- a/docs/Tutorials/MathComponent/Tutorial.md
+++ b/docs/Tutorials/MathComponent/Tutorial.md
@@ -1133,7 +1133,7 @@ and 10.
    This line tells the build system to make the unit test build
    depend on the `STest` build module.
 
-1. Add `#include STest/Random/Random.hpp` to `main.cpp`.
+1. Add `#include "STest/Random/Random.hpp"` to `main.cpp`.
 
 1. Add the following line to the `main` function of `main.cpp`,
    just before the return statement:

